linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_unique_name/rule.yml

```diff
@@ -2,9 +2,15 @@ documentation_complete: true
 
 title: 'Ensure All Accounts on the System Have Unique Names'
 
-description: 'Change usernames, or delete accounts, so each has a unique name.'
+description: |-
+    Although the <tt>useradd</tt> utility prevents creation of duplicate user
+    names, it is possible for a malicious administrator to manually edit the
+    <tt>/etc/passwd</tt> file and change the user name.
```

Comment:
This reads like a warning rather than a description to me which if you want to keep it I would suggest moving to a warning. Does the following description make more sense for users?

```
To ensure all accounts have unique names, run the following command:
<pre>$ sudo getent passwd | awk -F: '{ print $1}' | uniq -d</pre>
If a username is returned, change or delete the username.
```

Reply:
This looks like an OCIL clause too.

Reply:
@yuumasato you are right... although the verbs are a bit different. Suggestions?

Reply:
Current suggestion: bb1fbb95b43fb72531276ce71bcf24d993493261.
I didn't pay much attention to the verb though.